content/en/blog/_posts/2019-03-29-kube-proxy-subtleties-debugging-an-intermittent-connection-resets.md

```diff
@@ -133,6 +133,9 @@ connection.
 src="/images/blog/2019-03-26-kube-proxy-subtleties-debugging-an-intermittent-connection-resets/connection-reset-packet-flow.png"
 caption="Connection reset packet flow">}}
 
+Correction for previous diagram: Destination for packet 5 should be 10.0.1.2:80,
+not 10.0.0.1:80.
```

Comment:
If we were updating this, we might write:
```markdown
_This article was updated in April 2024 to note an error in the previous diagram:
the destination for packet 5 should be `10.0.1.2:80`, not `10.0.0.1:80`._
```

(If we were updating the article) it would also be OK to redraw the diagram and to use a different, appropriate note about the change. I'd use SVG if I were making that change, and I'd check with the original author that they were OK with having the article corrected.